src/include/packager/repositories_include.rb

```diff
@@ -483,11 +483,12 @@ def confirm_plain_repo
     # Check and install the packages needed for accessing the URL scheme.
     # @param scheme [String] URL scheme of the new repository
     def install_mount_package(scheme)
-      if scheme == "smb" && !File.exist?("/sbin/mount.cifs")
-        log.info("Installing missing 'cifs-mount' package...")
-        # install cifs-mount package
-        PackageSystem.CheckAndInstallPackages(["cifs-mount"])
-      end
+      return if scheme != "smb"
```

Comment:
NP: Just merging the lines?

```ruby
return if scheme != "smb" || File.exist?("/sbin/mount.cifs")
```

Reply:
I did it intentionally. I prefer simple exit criteria. For me it is two logical statement.

1. not samba
2. already installed

Reply:
ok